src/navigation/navigationhandler.cpp

```diff
@@ -100,6 +100,16 @@ namespace {
         "again.",
         openspace::properties::Property::Visibility::User
     };
+
+    constexpr openspace::properties::Property::PropertyInfo FadeJumpThresholdInfo = {
```

Comment:
Oh, do we really want a property for this?  It's a bit unclear that it is related to time, and it's not really related to the NavigationHandler at all.  **If we want to keep it, it should potentially be moved to the `TimeMananger` instead. ** To me, that seems like a more natural placement, since the function is located in `openspace.time`. Thoughts?  

However, IIRC we has a discussion of whether we even want this feature of automatically fading or not depending on time. It can seem a bit "magic" or confusing to our users.